launchpad-frontend/src/App.js

```diff
@@ -1,25 +1,19 @@
-import logo from "./logo.svg";
-import "./App.css";
+import Login from "./pages/login";
+import SignUp from "./pages/register";
+import './App.css';
+
+import {BrowserRouter as Router, Route, Routes} from "react-router-dom";
```

Comment:
I dont think react-router-dom stuff is included in the paclckage-lock. Can you run `npm install react-router-dom --save` in the terminal for this branch and then repush